crates/ruff_python_parser/Cargo.toml

```diff
@@ -22,6 +22,7 @@ bstr = { workspace = true }
 compact_str = { workspace = true }
 memchr = { workspace = true }
 rustc-hash = { workspace = true }
+serde = { workspace = true }
```

Comment:
We should make `serde` an optional dependency. Not all downstream users need serde serialization